src/GovUk.Education.ExploreEducationStatistics.Admin/Validators/ValidationMessages.cs

```diff
@@ -205,6 +205,20 @@ public static ErrorViewModel GenerateErrorDataReplacementAlreadyInProgress()
         };
     }
 
+    public static readonly LocalizableMessage DatasetTitleTooLong = new(
+        Code: nameof(DatasetTitleTooLong),
+        Message: "Subject title '{0}' must be 120 characters or less"
```

Comment:
I've made the messages reference the names of the fields on the front end to avoid confusion. I don't think we should alter terminology until a time when it can be changed across the board.